src/install/patch_install.zig

```diff
@@ -170,42 +170,23 @@ pub const PatchTask = struct {
         // TODO only works for npm package
         // need to switch on version.tag and handle each case appropriately
         const calc_hash = &this.callback.calc_hash;
-        const hash = switch (calc_hash.result orelse @panic("Calc hash didn't run, this is a bug in Bun.")) {
-            .result => |h| h,
-            .err => |e| {
-                if (e.getErrno() == bun.C.E.NOENT) {
-                    const fmt = "\n\n<r><red>error<r>: could not find patch file <b>{s}<r>\n\nPlease make sure it exists.\n\nTo create a new patch file run:\n\n  <cyan>bun patch {s}<r>\n";
-                    const args = .{
-                        this.callback.calc_hash.patchfile_path,
-                        manager.lockfile.patched_dependencies.get(calc_hash.name_and_version_hash).?.path.slice(manager.lockfile.buffers.string_bytes.items),
-                    };
-                    if (comptime log_level.showProgress()) {
-                        Output.prettyWithPrinterFn(fmt, args, Progress.log, &manager.progress);
-                    } else {
-                        Output.prettyErrorln(
-                            fmt,
-                            args,
-                        );
-                        Output.flush();
-                    }
-                    Global.crash();
-                }
-
-                const fmt = "\n\n<r><red>error<r>: {s}{s} while calculating hash for patchfile: <b>{s}<r>\n";
-                const args = .{ @tagName(e.getErrno()), e.path, this.callback.calc_hash.patchfile_path };
-                if (comptime log_level.showProgress()) {
-                    Output.prettyWithPrinterFn(fmt, args, Progress.log, &manager.progress);
-                } else {
-                    Output.prettyErrorln(
-                        fmt,
-                        args,
-                    );
-                    Output.flush();
-                }
-                Global.crash();
-
-                return;
-            },
+        const hash = calc_hash.result orelse {
+            const fmt = "\n\nErrors occured while calculating hash for <b>{s}<r>:\n\n";
+            const args = .{this.callback.calc_hash.patchfile_path};
+            if (comptime log_level.showProgress()) {
+                Output.prettyWithPrinterFn(fmt, args, Progress.log, &manager.progress);
+            } else {
+                Output.prettyErrorln(
+                    fmt,
+                    args,
+                );
+            }
+            if (calc_hash.logger.errors > 0) {
+                Output.prettyErrorln("\n\n", .{});
+                calc_hash.logger.printForLogLevel(Output.writer()) catch {};
+            }
+            Output.flush();
```

Comment:
`flush()` before crash can be deleted, we flush stdout/stderr in our `Bun_atexit` handler